.github/workflows/release.yml

```diff
@@ -4,7 +4,45 @@ on:
   push:
     branches: [master]
   pull_request:
+  workflow_dispatch:
 jobs:
+  check-changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Check if Chart.yaml was modified
+        id: check-chart
+        run: |
+          git fetch origin ${{ github.base_ref || 'master' }}
+
+          # Check if Chart.yaml was modified in this PR
+          if git diff --name-only origin/${{ github.base_ref || 'master' }}...HEAD | grep -q "charts/lakefs/Chart.yaml"; then
```

Comment:
Check if github.base_ref is set in case if 'push' to master. I'm fine if we verify on pull-request only.

Reply:
I think we should focus on pull-requests only. I didn't really consider the push to master scenario since we should catch this on the PR. Also if we do want to push force for some reason (override check) I don't want it to fail on master